src/pcb/pcb_silkscreen_pill.ts

```diff
@@ -0,0 +1,19 @@
+import { z } from "zod"
+import { distance } from "../units"
+import { layer_ref, visible_layer } from "./properties/layer_ref"
+import { point } from "src/common/point"
+
+export const pcb_silkscreen_pill = z
+  .object({
+    type: z.literal("pcb_silkscreen_pill"),
+    pcb_silkscreen_pill_id: z.string(),
+    pcb_component_id: z.string(),
+    center: point,
+    radius_x: distance,
+    radius_y: distance,
```

Comment:
i was wrong this is sufficient, sorry about that!!!

```suggestion
    width: distance,
    height: distance,
```